cirq-core/cirq/ops/gateset.py

```diff
@@ -54,6 +67,39 @@ class GateFamily:
             >>> assert cirq.Rx(rads=np.pi) in gate_family
             >>> assert cirq.X ** sympy.Symbol("theta") in gate_family
 
+    Tags can be used to add further constraints on containment checks using `tags_to_accept` and
```

Comment:
Sounds good. I will remove the code block explaining the semantics and replace it with your bullet list suggestion, but I will keep "If an `Operation` contains tags..." and "For the purpose of tag comparisons..." since they explain additional behaviors not covered by the above.

Reply:
I'm omitting ticks around "GateFamily" to be consistent with the docstring before this section, but let me know if you prefer including them. Will hyperlinking work properly in this case since it's referring to the class for which the docstring is defined?